db/migrate/0_database_structure.rb

```diff
@@ -0,0 +1,433 @@
+        t.string "program", limit: 255
+        t.integer "start_year", limit: 4
+        t.integer "user_id", limit: 4
+        t.string "avatar_file_name", limit: 255
+        t.string "avatar_content_type", limit: 255
+        t.integer "avatar_file_size", limit: 4
+        t.datetime "avatar_updated_at"
+        t.datetime "created_at"
+        t.datetime "updated_at"
+        t.integer "first_post", limit: 4
+        t.string "email", limit: 255
+        t.string "stil_id", limit: 255
+        t.string "phone", limit: 255
+        t.string "lastname", limit: 255
+      end
+    end
+    unless table_exists? :rents
+      create_table "rents", force: :cascade do |t|
+        t.datetime "d_from"
+        t.datetime "d_til"
+        t.string "name", limit: 255
+        t.string "lastname", limit: 255
+        t.string "email", limit: 255
+        t.string "phone", limit: 255
+        t.text "purpose", limit: 65535
+        t.boolean "disclaimer", limit: 1, default: false
+        t.boolean "aktiv", limit: 1, default: true
+        t.integer "council_id", limit: 4
+        t.integer "profile_id", limit: 4
+        t.datetime "created_at"
+        t.datetime "updated_at"
+        t.text "comment", limit: 65535
+        t.string "status", limit: 255, default: "Ej bestÃƒÂ¤md"
+        t.boolean "service", limit: 1, default: false
+        t.string "access_code", limit: 255
+      end
+    end
+    unless table_exists? :short_links
+      create_table "short_links", force: :cascade do |t|
+        t.string "link", limit: 255, null: false
+        t.text "target", limit: 65535, null: false
+      end
+      add_index "short_links", ["link"], name: "index_short_links_on_link", using: :btree
+    end
+    unless table_exists? :subcategories
+      create_table "subcategories", force: :cascade do |t|
+        t.string "text", limit: 255
+        t.datetime "created_at"
+        t.datetime "updated_at"
+      end
+    end
+    unless table_exists? :users
+      create_table "users", force: :cascade do |t|
+        t.string "username", limit: 255, null: false
+        t.string "email", limit: 255, default: "", null: false
+        t.string "encrypted_password", limit: 255, default: "", null: false
+        t.string "reset_password_token", limit: 255
+        t.datetime "reset_password_sent_at"
+        t.datetime "remember_created_at"
+        t.integer "sign_in_count", limit: 4, default: 0, null: false
+        t.datetime "current_sign_in_at"
+        t.datetime "last_sign_in_at"
+        t.string "current_sign_in_ip", limit: 255
+        t.string "last_sign_in_ip", limit: 255
+        t.integer "role_id", limit: 4, default: 2, null: false
+        t.datetime "created_at"
+        t.datetime "updated_at"
+      end
+      add_index "users", ["email"], name: "index_users_on_email", unique: true, using: :btree
+      add_index "users", ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true, using: :btree
```

Comment:
Line is too long. [123/100]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.